ciao-image/datastore/datastore_test.go

```diff
@@ -51,27 +54,29 @@ func testGetAll(t *testing.T, d RawDataStore, m MetaDataStore) {
 		State: Created,
 	}
 
-	cache := ImageCache{}
-	cache.Init(d, m)
+	imageStore := ImageStore{}
+	_ = imageStore.Init(d, m)
 
 	// create the entry
-	err := cache.CreateImage(i)
+	err := imageStore.CreateImage(i)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	// retrieve the entry
-	images, err := cache.GetAllImages()
+	images, err := imageStore.GetAllImages()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	if len(images) != 1 {
-		t.Fatalf("len is actually %d\n", len(images))
-	}
+	if fmt.Sprintf("%T", m) != "*datastore.Noop" {
```

Comment:
This is a bit odd.  Is the intention to simply check to see if m is not a *datastore.Noop?  if so it might be better to use a type assertion.

if _, ok := m.(*datastoreNoop); !ok {

}